README.md

```diff
@@ -1,5 +1,8 @@
 # Recommendations
 
+[![Build Status](https://travis-ci.org/nyudevops-recommendation/recommendations.svg?branch=master)](https://travis-ci.org/nyudevops-recommendation/recommendations)
+[![codecov](https://codecov.io/gh/qijinze911/recommendations/branch/master/graph/badge.svg)](https://codecov.io/gh/qijinze911/recommendations)
```

Comment:
for the [codecov] you should also use the link for the organization as the [build status], or it shows unknown on other people's sides